database/migrations/2016_02_17_195543_create_user_waiting_room_pivot_table.php

```diff
@@ -14,9 +14,9 @@ public function up()
     {
         Schema::create('user_waiting_room', function (Blueprint $table) {
             $table->string('user_id');
-            $table->foreign('user_id')->references('id')->on('users');
+            $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
             $table->integer('waiting_room_id')->unsigned();
-            $table->foreign('waiting_room_id')->references('id')->on('waiting_rooms');
+            $table->foreign('waiting_room_id')->references('id')->on('waiting_rooms')->onDelete('cascade');
```

Comment:
I don't think it's a big deal to wipe and restore the DBs for now... and just start adding custom update migrations post deploy to server